src/it/scala/inox/solvers/unrolling/MapMergeSuite.scala

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2009-2020 EPFL, Lausanne */
+
+package inox
+package solvers
+package unrolling
+
+class MapMergeSuite extends SolvingTestSuite with DatastructureUtils {
+  import trees._
+  import dsl._
+
+  override def configurations = for {
+    solverName   <- Seq("nativez3", "unrollz3", "smt-z3")
```

Comment:
I'm surprised none of the tests in this file work with the encoding. Did you take a look at why they were failing?

Reply:
They all return `Unknown` from what I remember (both in CVC4 and Princess).